packages/maskbook/tsconfig.json

```diff
@@ -12,7 +12,6 @@
   "references": [
     { "path": "../shared" },
     { "path": "../web3-shared" },
-    { "path": "../constants" },
```

Comment:
Why not this?
```suggestion
    { "path": "../web3-constants" },
```

Reply:
maskbook not need constants

https://github.com/DimensionDev/Maskbook/blob/e872ba1643a3e43f130088456551119eccc05a37/packages/maskbook/package.json#L21-L29